src/libraries/System.Private.CoreLib/src/System/BitConverter.cs

```diff
@@ -616,6 +617,7 @@ public static float ToSingle(ReadOnlySpan<byte> value)
         /// <param name="value">A read-only span containing the bytes to convert.</param>
         /// <returns>A double-precision floating-point value representing the converted bytes.</returns>
         /// <exception cref="ArgumentOutOfRangeException">The length of <paramref name="value"/> is less than 8.</exception>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
There are many other similar methods in this file: ToSingle, ToHalf, ... . Do all of them need this?

Reply:
Yes, we should also update the Single/Half versions, at a minimum.